helper/testhelpers/testhelpers.go

```diff
@@ -3,9 +3,12 @@ package testhelpers
 import (
 	"context"
 	"encoding/base64"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/vault/builtin/credential/ldap"
```

Comment:
Maybe it's only me, but I prefer the old ordered import order.

Reply:
Fixed.